ggalluvial_cheatsheet.Rmd

```diff
@@ -0,0 +1,89 @@
+# ggalluvial Cheatsheet
+
+# Meggie Wen
```

Comment:
Single hashtag cannot be used in the file except the first line. (If there are, new chapters will be created.)